dart/test/keyword_test_keyword_test_generated.dart

```diff
@@ -0,0 +1,280 @@
+// automatically generated by the FlatBuffers compiler, do not modify
+// ignore_for_file: unused_import, unused_field, unused_element, unused_local_variable
+
+library keyword_test;
+
+import 'dart:typed_data' show Uint8List;
+import 'package:flat_buffers/flat_buffers.dart' as fb;
+
+
+class ABC {
+  final int value;
+  const ABC._(this.value);
+
+  factory ABC.fromValue(int value) {
+    final result = values[value];
+    if (result == null) {
+        throw StateError('Invalid value $value for bit flag enum ABC');
+    }
+    return result;
+  }
+
+  static ABC? _createOrNull(int? value) => 
+      value == null ? null : ABC.fromValue(value);
+
+  static const int minValue = 0;
+  static const int maxValue = 2;
+  static bool containsValue(int value) => values.containsKey(value);
+
+  static const ABC void_ = ABC._(0);
+  static const ABC where = ABC._(1);
+  static const ABC stackalloc = ABC._(2);
+  static const Map<int, ABC> values = {
+    0: void_,
+    1: where,
+    2: stackalloc};
+
+  static const fb.Reader<ABC> reader = _ABCReader();
+
+  @override
+  String toString() {
+    return 'ABC{value: $value}';
+  }
+}
+
+class _ABCReader extends fb.Reader<ABC> {
+  const _ABCReader();
+
+  @override
+  int get size => 4;
+
+  @override
+  ABC read(fb.BufferContext bc, int offset) =>
+      ABC.fromValue(const fb.Int32Reader().read(bc, offset));
+}
+
+class public {
```

Comment:
Class names should always start with an uppercase letter, or `$` at worst. This will cause all sorts of code highlighting issues and analysis complaints.

Reply:
I could make types `kUpperCamel` case rather than `kKeep` which would fix this problem. It doesn't affect any of the old generated files in our repo, but it'll likely break users. My 2c is that this PR should focus on applying the Namer class and focus on compatibility. We can phase in the style changes later.

Reply:
I'd be pretty surprised if any users were depending on this behavior, since it would almost certainly cause analysis issues in their repo and code highlighting issues.